src/v/cluster/health_monitor_types.cc

```diff
@@ -312,7 +362,7 @@ adl<cluster::node_health_report>::from(iobuf_parser& p) {
     auto redpanda_version = adl<cluster::node::application_version>{}.from(p);
     auto uptime = adl<std::chrono::milliseconds>{}.from(p);
     auto disks = adl<std::vector<storage::disk>>{}.from(p);
-    auto topics = adl<std::vector<cluster::topic_status>>{}.from(p);
+    auto topics = adl<ss::chunked_fifo<cluster::topic_status>>{}.from(p);
```

Comment:
was this (and the hunk below) because this adl wasn't present in 23.1.x?

Reply:
Yes adl wasn't present in dev when I made my patch